server/engine/telemetry.go

```diff
@@ -34,6 +35,24 @@ func BuildSegmentPropertiesMap(db *gorm.DB) analytics.Properties {
 	return propertiesMap
 }
 
+func StoreMetricFromMainOutputs(db *gorm.DB) {
+
+	//var outputsMap map[string]interface{}
+	var mainOutput model.Output
+	db.Where("module_name = ?", "").Find(&mainOutput)
+
+	if loc, exists := mainOutput.Values["location"]; exists {
+		model.SetMetric(db, model.Region, loc.(map[string]interface{})["value"].(string))
+	} else {
+		log.Errorf("Location of deployment is missing : will not be included in telemetry")
```

Comment:
Should be log.Error w/ no formatting template included.

Reply:
Updated